app/services/UploadService.scala

```diff
@@ -23,4 +27,11 @@ class UploadService @Inject() (val graphqlConfiguration: GraphQLConfiguration)(i
     val variables = afam.Variables(addFileAndMetadataInput)
     sendApiRequest(addFilesAndMetadataClient, afam.document, token, variables).map(data => data.addFilesAndMetadata)
   }
+
+  def uploadDraftMetadata(bucket: String, key: String, draftMetadata: String): IO[CompletedUpload] = {
```

Comment:
This should have some unit tests

Reply:
@ian-hoyle Add unit tests to the `UploadServiceSpec.scala` for this new method.

Reply:
tests added